src/Microsoft.Unity.Analyzers.Tests/SetLocalPositionAndRotationTests.cs

```diff
@@ -27,6 +27,11 @@ void Update()
 }
 ";
 
+		var method = GetCSharpDiagnosticAnalyzer().ExpressionContext.SetPositionAndRotationMethodName;
+		var type = typeof(UnityEngine.Transform);
+
+		Skip.IfNot(MethodExists(type.FullName!, method, type.Namespace!), $"This Unity version does not support {type}.{method}");
```

Comment:
using `type.Namespace` to get the assembly name for filtering looks weird, let's be explicit and say something like `type.Assembly.GetName().Name`.

Reply:
`type.Assembly.GetName().Name` is the analyzer assembly given we use stubs, going to use "UnityEngine" :)